client/components/payment-activity/style.scss

```diff
@@ -23,6 +23,19 @@
 				padding: 16px 0 19px;
 			}
 		}
+
+		&__header {
+			.wcpay-filter.components-custom-select-control {
+				.wcpay-filter.components-custom-select-control__menu {
+					max-height: fit-content;
+				}
+			}
+
+			@include breakpoint( '<660px' ) {
+				flex-direction: column !important;
+				gap: 16px;
+			}
```

Comment:
Suggest adding a comment here explaining the intention for future developers who might need to tweak or work around this decision.

Reply:
Added a comment 👍